cos_registration_server/applications/models.py

```diff
@@ -43,3 +45,55 @@ class FoxgloveDashboard(Dashboard):  # noqa: DJ08
     uid: Unique ID of the dashboard.
     dashboard: Dashboard JSON.
     """
+
+
+class AlertRule(models.Model):
```

Comment:
Maybe `AlertRulesFile` or simply `RulesFile` @artivis any opinion?

Reply:
yeah but a dashboard file contains a dashboard and that it.
a Rules file contain groups of alert rules.